wiki/Tournaments/CWC/1/en.md

```diff
@@ -0,0 +1,265 @@
+---
+tags:
+- CWC
+---
+# Catch the Beat World Cup
+
+The **Catch the Beat World Cup** (***CWC***) was a team-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It was the 1st installment of the osu!catch World Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2011-09-30/2011-11-01 |
+| Round 1 | 2011-11-04/2011-11-06 |
+| Round 2 | 2011-11-11/2011-11-13 |
+| Round 3 | 2011-11-18/2011-11-20 |
+| Round 4 | 2011-11-25/2011-11-27 |
+| Round 5 | 2011-12-02/2011-12-04 |
+| Quarterfinals | 2012-01-07/2011-01-08 |
+| Semifinals | 2012-01-14/2011-01-15 |
+| Finals | 2012-02-05 |
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
+
+*Note: 1 month of osu!supporter tag also given to backup member of the winning team*
```

Comment:
1 month of osu!supporter tag was also given to backup member of the winning team